runtime/tests/automl_runtime/sklearn/one_hot_transformer_test.py

```diff
@@ -0,0 +1,126 @@
+#
+# Copyright (C) 2021 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import unittest
+
+import pandas as pd
+import numpy as np
+from scipy import sparse
+from databricks.automl_runtime.sklearn import OneHotTransformer
+from sklearn.pipeline import Pipeline
+
+class TestOneHotEncoder(unittest.TestCase):
+    
+    def setUp(self) -> None:
+        self.numeric_X = [[1,2,3,4]]
+        # The expected output for sklearn.OneHotEncoder is [1,1,1,1] (so a difference)
+        self.expected_numeric_x = [[1,2,3,4]]
+
+        self.categorical_X = [['Male', 1], ['Female', 3], ['Female', 2]]
+        # the numerical value is the last column (unsure if we can change that)
+        self.expected_categorical_X_value = [[1, 0, 1], [0, 1, 3], [0, 1, 2]]
+        self.expected_categorical_X_indicator = [[1, 0, 0, 1], [0, 1, 0, 3], [0, 1, 0, 2]]
+
+        self.categorical_X_with_none = [['Male', 1], [np.nan, 2], [None, 3], [pd.NA, 1]]
+        # np.nan == None, but pd.na is a different item (unknown?) so we fill a different column for both
```

Comment:
I think we can remove these comments, they seem to be about a impl details that aren't super necessary to note